code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -307,6 +316,24 @@ GLOBAL_LIST_EMPTY(PDAs)
 //BASIC FUNCTIONS===================================
 
 			if("Refresh")//Refresh, goes to the end of the proc.
+			if ("Toggle_Font")
+				if (font_index == 3)
+					font_index = 0
+				else
+					font_index++

```

Comment:
Replace this whole if statement with
```
font_index++
font_index = font_index%4
```

Reply:
@lordpidey Did I do it right?

Reply:
I think that might have broken it.

Reply:
No, you need to replace lines 320 through 323 with what I posted.  You also took out 319.

Reply:
Oh whoops

Reply:
Okay it should work now.

Reply:
@lordpidey I did the thing